docs/source/faq.rst

```diff
@@ -9,6 +9,8 @@ FAQ
 I can't seem to access my HackRF under Linux
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. _faq_hackrf_under_linux:
```

Comment:
Can you please place this anchor above its heading rather than below? This allows the heading to be seen when a link to the anchor is followed.